utils/client.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+"""WCB CLI client.
+"""
+
+from __future__ import print_function
+
+import argparse
+import json
+import os
+import urllib2
+import urlparse
+
+ENTITIES = {
+    '/': ['', ('',), (), ()],
+    'achievement': ['achievement', ('', 'id'), ('name', 'distance'), ('id',)],
+    'achievements': ['achievements', ('team', 'achievement'), ('team_id', 'achievement_id'),
+                     ('id',)],
+    'cause': ['causes', ('', 'id'), ('name',), ('id',)],
+    'donor': ['donor', (), (), ()],
+    'donors': ['donors', (), (), ()],
+    'event': ['events', ('', 'id'), ('name', 'description', 'start_date', 'end_date', 'team_limit',
+                                     'team_building_start', 'team_building_end'), ('id',)],
+    'locality': ['localities', ('', 'id'), ('name',), ('id',)],
+    'participant': ['participants', ('fbid',), ('fbid',), ('fbid',)],
+    'record': ['records', ('id',), ('date', 'distance', 'participant_id', 'source_id'), ()],
+    'source': ['sources', ('', 'id'), ('name',), ('id',)],
+    'sponsor': ['sponsor', (), (), ()],
+    'sponsors': ['sponsors', (), (), ()],
+    'team': ['teams', ('', 'id'), ('name',), ('id',)],
+}
+
+CRUD = {
+    'c': 'POST',
+    'r': 'GET',
+    'u': 'PATCH',
+    'd': 'DELETE',
+}
+
```

Comment:
I think that we also want something like:

    (CREATE, READ, UPDATE, DELETE) = range(3)

This will make the entity indexing below nicer.